src/Test/TestProductionClearing.map

```diff
@@ -1,4 +1,4 @@
- 1FC  GR  GR  GR
+ 1FC  GR  GR  CT
```

Comment:
Odd change, considering there are no other test changes. Is this just... for orientation purposes?

Reply:
Travel objectives were being added even when unreachable.
I fixed *that* bug, but that meant that the unit on the factory wouldn't move since it had nowhere useful to go.
I figured that's not really a representative use case, so I modified the test instead.